python/lsst/pipe/base/pipelineBuilder.py

```diff
@@ -55,123 +50,83 @@
 
-    def __init__(self, taskFactory):
-        self.taskFactory = taskFactory
+    def pipeline(self, order_pipeline=False):
+        """Return updated pipeline instance.
 
-    def makePipeline(self, args):
-        """Build pipeline from command line arguments.
+        Pipeline will be checked for possible inconsistencies before
+        returning, exception is raised if any problems are detected.
 
         Parameters
         ----------
-        args : `argparse.Namespace`
-            Parsed command line.
+        order_pipeline : `bool`, optional
+            If `True` then order resulting pipeline according to Task data
+            dependencies.
 
         Returns
         -------
-        `Pipeline` instance.
-
-        Raises
-        ------
-        Exceptions will be raised for any kind of error conditions.
+        pipeline : `Pipeline`
         """
-
-        # need instrument/package name to find overrides
-        obsPkg = None
-        instrument = None
-        if args.instrument_overrides:
-            # mapperClass = dafPersist.Butler.getMapperClass(args.input)
-            # instrument = mapperClass.getCameraName()
-            # obsPkg = mapperClass.getPackageName()
-            pass
-
-        if args.pipeline:
-
-            # load it from a pickle file, will raise on error
-            with open(args.pipeline, 'rb') as pickleFile:
-                pipeline = pickle.load(pickleFile)
-
-            # check type
-            if not isinstance(pipeline, Pipeline):
-                msg = "Pickle file `{}' contains something other than Pipeline"
-                raise TypeError(msg.format(args.pipeline))
-
-        else:
-            # start with empty one
-            pipeline = Pipeline()
-
-        # loop over all pipeline actions and apply them in order
-        for action in args.pipeline_actions:
-
-            if action.action == "new_task":
-
-                self._newTask(pipeline, action.value, action.label, obsPkg, instrument)
-
-            elif action.action == "delete_task":
-
-                self._dropTask(pipeline, action.label)
-
-            elif action.action == "move_task":
-
-                self._moveTask(pipeline, action.label, int(action.value))
-
-            elif action.action == "relabel":
-
-                self._labelTask(pipeline, action.label, action.value)
-
-            elif action.action == "config":
-
-                self._configOverride(pipeline, action.label, action.value)
-
-            elif action.action == "configfile":
-
-                self._configOverrideFile(pipeline, action.label, action.value)
-
         # conditionally re-order pipeline if requested, but unconditionally
         # check for possible errors
-        orderedPipeline = pipeTools.orderPipeline(pipeline, self.taskFactory)
-        if args.order_pipeline:
-            pipeline = orderedPipeline
-
-        return pipeline
+        orderedPipeline = pipeTools.orderPipeline(self._pipeline, self._taskFactory)
```

Comment:
Missing import for `pipeTools`?

Reply:
I think it should be there, unit test would fail without that import

Reply:
I suspect it was fixed on a later commit and I didn't notice.